### PR TITLE
Add a test of block label redefinition.

### PR DESCRIPTION
**Pull request by JSStats**
To resolve https://github.com/WebAssembly/spec/issues/210

### CODE REVIEW

File: ml-proto/test/labels.wast [hunk 1]
```diff
@@ -99,6 +99,13 @@
     )
   )
 
+  (func $redefinition (result i32)
+    (block $l1
+      (i32.add (block $l1
+		 (i32.const 2))
```
Comment (rossberg-chromium) [inline ml-proto/test/labels.wast]: Nit: formatting seems off.

Comment (JSStats) [inline ml-proto/test/labels.wast]: There is no style guide for the formatting of s-exp. Putting trailing braces on separate lines is what I see beginners do and once people have a little experience they prefer the increased information density of placing them on one line - you will not find experienced lisp codes putting closing braces on separate lines as a style rule (there are some exceptions, but not for something so simple as above).

Comment (rossberg-chromium) [inline ml-proto/test/labels.wast]: I was referring to the inconsistent indentation and the extra line breaks in particular. That said, it's good practice for changes to adapt to surrounding styles. ;)

**Comment by sunfishcode** [comment]
lgtm

**Comment by sunfishcode** [comment]
This branch needs a merge conflict resolution.

**Comment by rossberg-chromium** [comment]
Ping @JSStats, can you resolve the merge conflict?

**Comment by JSStats** [comment]
ptal

**Pull request merged**